packages/core/src/api.js

```diff
@@ -115,6 +115,10 @@ export function createPercyServer(percy, port) {
     .route('post', '/percy/flush', async (req, res) => res.json(200, {
       success: await percy.flush(req.body).then(() => true)
     }))
+  // creating driverWrapper.js
+    .route('post', '/percy/driverWrapper', async (req, res) => res.json(200, {
```

Comment:
need better name for the endpoint

Reply:
Named as `automateScreenshot` .
Let me know if this requires changes.